### PR TITLE
Automation test release 3

### PR DESCRIPTION
**Pull request by gokul-sol**
### Jira link (if applicable)
Screenshot of PR passing consistently before pushing to Master

Screenshot 2024-08-13 at 14 09 23

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by pats-john — APPROVED** [review]

**Review by sanjayparekh-hmcts — DISMISSED** [review]

**Review by sanjayparekh-hmcts — APPROVED** [review]

**Review by pats-john — APPROVED** [review]

**Review by nitinprabhuhmcts — APPROVED** [review]

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -74,6 +86,28 @@ withPipeline("nodejs", product, component) {
             env.IDAM_OAUTH2_CLIENT_ID='sscs'
     }
 
+     afterSuccess('smoketest:preview') {
```
Comment (nitinprabhuhmcts) [inline Jenkinsfile_CNP:89]: @gokul-sol  why are we introducing playwright tests in preview when we know they are not stable consistently ? @sanjayparekh-hmcts  you aware of this ?

Comment (gokul-sol) [reply Jenkinsfile_CNP:89]: @nitinprabhuhmcts this is a placeholder step which enables us to check if tests are running successfully against AAT in Jenkins before pushing it to Master. We will remove this step before pushing the code.

**Review by nitinprabhuhmcts — COMMENTED** [review]
Discussed with @gokul-sol @sanjayparekh-hmcts

**Review by nitinprabhuhmcts — APPROVED** [review]

**Review by sanjayparekh-hmcts — APPROVED** [review]
As discussed with Nitin and Gokul

**Pull request merged**